unblob/models.py

```diff
@@ -25,11 +33,19 @@
 class Task:
     path: Path
     depth: int
-    chunk_id: str
+    extractable_id: str
```

Comment:
With this PR, we could think of the extraction source as either a `ValidChunk` or a `MultiFile`. Previously it was just `ValidChunk`, so `chunk_id` was appropriate.

Reply:
Further thinking of *extraction source*, there is another one, that is the middle ground here: `WholeFile`.
It is currently handled as a special, degenerate case of `ValidChunk`, with some special handling, but maybe it also deserves a distinct type, and separate processing.

Reply:
`WholeFile` is definitely out of scope for this PR, maybe a follow-up.

Reply:
I would stick with extractable_id unless we rename Extractable as well. WholeFile can be a special case of a ValidChunk, though not sure if that would make much difference.

Reply:
I do not like extraction source, I would rather focus from  a perspective that we want to identify something that represent some content, either inside a file (chunk) or a set of files. The fact that it could be extracted is a secondary thing. From this perspective both ExtractionSource and Extractable are both miss-fit.

I was thinking along the line like: (data)block, (data)blob, content, though not happy with those either.

Reply:
And the winner is Blob ;)